### ISSUE TITLE
0.4.0 build error with "includeHighChartsMore: true"

### ISSUE THREAD

**Issue by denchen** [ahmadsoe/ember-highcharts#58]
I tried upgrading from 0.2.1 to 0.4.0 but get a build error which I narrowed it down to including the highcharts-more addon.

```
% ember build
DEPRECATION: Node v0.10.40 is no longer supported by Ember CLI. Please update to a more recent version of Node
undefined
version: 1.13.15
Build failed.
ENOENT, no such file or directory '/home/denchen/git/dashboard/tmp/concat_with_maps-input_base_path-5vvK3bT3.tmp/0/vendor/highcharts/highcharts/highcharts-more.src.js'
Error: ENOENT, no such file or directory '/home/denchen/git/dashboard/tmp/concat_with_maps-input_base_path-5vvK3bT3.tmp/0/vendor/highcharts/highcharts/highcharts-more.src.js'
    at Object.fs.statSync (fs.js:696:18)
    at ConcatWithMaps.keyForFile (/home/denchen/git/dashboard/node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/broccoli-caching-writer/index.js:90:20)
    at Array.map (native)
    at ConcatWithMaps.CachingWriter._conditionalBuild (/home/denchen/git/dashboard/node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/broccoli-caching-writer/index.js:112:65)
    at /home/denchen/git/dashboard/node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/broccoli-caching-writer/node_modules/broccoli-plugin/read_compat.js:61:34
    at lib$rsvp$$internal$$tryCatch (/home/denchen/git/dashboard/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:493:16)
    at lib$rsvp$$internal$$invokeCallback (/home/denchen/git/dashboard/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:505:17)
    at lib$rsvp$$internal$$publish (/home/denchen/git/dashboard/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:476:11)
    at Object.lib$rsvp$asap$$flush [as _onImmediate] (/home/denchen/git/dashboard/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1198:9)
    at processImmediate [as _immediateCallback] (timers.js:363:15)
```

HighCharts in Ember works just fine if I don't include `highcharts-more`. Unfortunately, I do need that package as I'm using boxplots, which are defined in `highcharts-more`

**Comment by denchen** [comment]
I believe I tracked down the problem. In `index.js` line 31, instead of:

`app.import(highchartsPath + '/highcharts/highcharts-more.src.js');`

It should be:

`app.import(highchartsPath + 'highcharts-more.src.js');`

When I made that change and rebuilt my app, all my charts are working again. I believe the same change needs to be made for line 35 as well for 3D charts.

I can make a pull request for this.
